### ISSUE TITLE
[Suggestion] Moving Item

### ISSUE THREAD

**Issue by 4Magma** [jaquadro/StorageDrawers#237]
If i realize that a item in a 1x2 or 2x2 drawer is there on the wrong place (wrong projected or
pumped in a by mistake unlocked drawer) i must dolly the drawer away from the 
network, unlock them and extract all the wrong item. 

Suggestion:
- a key like item, which is used to start internal transfers
- with the item shift right click on a target drawer section (is now priorized target for this item)
- and then right click on each source drawer section (same or oredict-same item)
- transfered 1 stack per second
- lock status is ignored, the drawer section is after the transfer empty

**Comment by Gabbleducky** [comment]
YES! I have had a lot of occasions where this would be useful, particularly when moving items to bigger drawers!
